From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Wed, 30 Jul 2025 15:35:45 -0500
Subject: [PATCH] use bash for dconf script

Seems to use bashisms, and /bin/sh is dash on Aeryn.

---
 data/dconf/make-dconf-override-db.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/dconf/make-dconf-override-db.sh b/data/dconf/make-dconf-override-db.sh
index 32cb1530..963af9cb 100755
--- a/data/dconf/make-dconf-override-db.sh
+++ b/data/dconf/make-dconf-override-db.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
 
 set -e
 
